### PR TITLE
Add cards example.

### PR DESCRIPTION
**Pull request by dvc94ch**

### CODE REVIEW

**Comment by alexjg** [comment]
I like this example and I think that at some point we should probably put something like the `Automerge` struct in the library proper. 

It's probably worth removing some or all of the comments which (I think) are left over from the JS version of this example. They're confusing as the API is a bit different.

File: automerge/Cargo.toml [hunk 1]
```diff
@@ -34,6 +34,7 @@ tracing-subscriber = {version = "0.2", features = ["chrono", "env-filter", "fmt"
 unicode-segmentation = "1.7.1"
 maplit = "^1.0.2"
 smol_str = "0.1.17"
+anyhow = "1.0.41"
```
Comment (alexjg) [inline automerge/Cargo.toml:37]: This should be an example only dependency.

Comment (jeffa5) [reply automerge/Cargo.toml:37]: This is already in the `dev-dependencies` section, I'm not aware of an examples only one?

Comment (alexjg) [reply automerge/Cargo.toml:37]: Ah, right you are, I assumed you could have dependencies scoped by target. `dev-dependencies` seems fine.

**Comment by alexjg** [comment]
Last nit, can you add a `Default` implementation for the `Automerge` struct to satisfy clippy?

**Pull request merged**